hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -964,6 +965,11 @@ bool MDMParser::init(DevStatus* status)
             sendFormated("AT+UMNOPROF=%d\r\n", UBLOX_SARA_UMNOPROF_SIM_SELECT);
             waitFinalResp(); // Not checking for error since we will reset either way
             goto reset_failure;
+        }
+		// For signal strength RSRP/RSRQ values on R410M
+        sendFormated("AT+UCGED=5\r\n");
+        if (RESP_OK != waitFinalResp()) {
```

Comment:
For `AT+UCGED?` query to work, we need to set `AT+UCGED=5` every time the radio powers up.

Reply:
Would it potentially make sense to run it every time before querying `AT+UCGED` like what we do in `getCellularGlobalIdentity()` with `AT+COPS=3,2`?

Reply:
We can also do that, but it _needs_ to be run only when the modem is ON.

Reply:
Me being OCD, so adding `UCGED=5` every time before calling `UCGED?`